charts/timescaledb-single/templates/statefulset-timescaledb.yaml

```diff
@@ -34,7 +34,7 @@ spec:
     spec:
       serviceAccountName: {{ template "timescaledb.serviceAccountName" . }}
       securityContext:
-        # The postgres user inside the TimescaleDB image has uid=1000.
+        # The postgres  user inside the TimescaleDB image has uid=1000.
```

Comment:
This is to trigger the job to run a Kind clutser, will be removed in the final commit